environments/common/ecr/ecr.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_kms_key" "ecr_kms" {
+  description         = "ECR KMS key"
+  enable_key_rotation = true
+}
+
+
+# tfsec:ignore:aws-ecr-enforce-immutable-repository
+resource "aws_ecr_repository" "trade-tariff-ecr_repo" {
+  for_each = toset([
+    "trade-tariff-frontend-ecr-${var.environment}",
+    "trade-tariff-backend-ecr-${var.environment}",
+    "trade-tariff-dutycal-ecr-${var.environment}",
```

Comment:
I suppose, we could probably just have one repository per environment, and put all the application images in it. Then it would be `trade-tariff/frontend:latest`, `trade-tariff/backend:latest`, and so on. That might be a bit of a nicer namespace (and mean less resources overall).

What do you think?